server/routes/teamAPI.js

```diff
@@ -0,0 +1,73 @@
+const express = require("express");
+const db = require("../db/models");
+const auth = require("../middleware/auth");
+
+const router = express.Router();
+
+// CREATE team
+// req.body: { name: name, members: [email:string] }
+router.post("/api/team/create", auth, async (req, res) => {
+  let team;
+  const findMembers = req.body.members.map((email) => {
+    return db.User.findOne({ email: email });
```

Comment:
we can use db.User.find({eamil:{$in: req.body.members}}) instead of map

Reply:
Wish I knew about this before I made it an array of promises 🙃